src/Components/Backoffice/BackofficeHeader.js

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react';
+import {styled,useTheme} from '@mui/material/styles';
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronRight from '@mui/icons-material/ChevronRight';
+import ChevronLeft from '@mui/icons-material/ChevronLeft';
+import {IconButton,
+        Box,
+        Typography,
+        Drawer,
+} from '@mui/material';
+import MuiAppBar from '@mui/material/AppBar';
+
+const BackofficeHeader = () => {
+
+  const drawerWidth = 240;
+
+  const DrawerHeader = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
+    ...theme.mixins.toolbar,
+    justifyContent: 'flex-end',
+  }));
+  const AppBar = styled(MuiAppBar, {
+    shouldForwardProp: (prop) => prop !== 'open',
+    })(({ theme, open }) => ({
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    ...(open && {
+      width: `calc(100% - ${drawerWidth}px)`,
+      marginLeft: `${drawerWidth}px`,
+      transition: theme.transitions.create(['margin', 'width'], {
+        easing: theme.transitions.easing.easeOut,
+        duration: theme.transitions.duration.enteringScreen,
+      }),
+    }),
+  }));
```

Comment:
move this components to a file for each of them on Utils components folder

Reply:
Solved!